src/characteristics/types.ts

```diff
@@ -0,0 +1,15 @@
+import type {
+  Characteristic as CharacteristicClass,
+  CharacteristicProps,
+} from 'homebridge';
+import type { MarkOptional } from 'ts-essentials';
+
+declare class DefaultCharacteristicClass extends CharacteristicClass {
+  constructor(
+    displayName: string,
+    UUID: string,
+    props?: MarkOptional<CharacteristicProps, 'format' | 'perms'>
```

Comment:
Created `DefaultCharacteristicClass` to be able not to pass `format` and `perms`